app/controllers/api/v1/bikes_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Api::V1::BikesController < Api::V1::BaseController
+  skip_before_action: verify_authencity_token, only: [:create, :update, :destroy]
+
+  def index
+    @bike = Bikes.all
+    # render json: @bikes
+  end
+
+  def show

```

Comment:
we need to delete (or comment) this method

Reply:
ok done.